File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1159.c
#include <stdio.h>

int main(){
    int n, i, soma;
    while(1){
        scanf("%d", &n);
        if(n==0){
            break;
        }
        if(n%2!=0){
            n++;
        }
        soma=0;
        for(i=0;i<5;i++){
            soma+=n;
            n+=2;
        }
        printf("%d\n", soma);
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1159.rb
while(true) do
    n = gets.to_i
    if(n==0) then
        break
    end
    if(n%2!=0) then
        n=n+1
    end
    soma=0
    for i in 1..5 do
        soma=soma+n
        n=n+2
    end
    puts(soma)
end